helm/README.md

```diff
@@ -111,22 +111,24 @@ providers, we recommend you use it. If you need to be able to deploy pods quickl
 If you are operating on a fresh persistent volume or similar, you may have to wait a little while for the model to download. You can check the status of your deployment by running
 
 ```bash
-kubectl get pods -n inference-ms
+kubectl get pods -n nim
 ```
 And check that the pods have become "Ready".
 
 Once that is true, you can try something like:
 
 ```bash
-helm -n inference-ms test my-nim --logs
+helm -n nim test my-nim --logs
 ```
 
 Which will run some simple inference requests. If the three tests pass, you'll know the deployment was successful.
 
+Setting up external ingress without adding an authentication layer is not recommended because NIM doesn't provide authentication on its own. The chart provides options for basic ingress, if desired.
```

Comment:
```suggestion
Avoid setting up external ingress without adding an authentication layer. This is because NIM doesn't provide authentication on its own. The chart provides options for basic ingress.
```